pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,94 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryBasicUserRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create"),
```

Comment:
if this is a spin-off of the basic user scoped to the registry then I think you need to list projectrequests.

A project request seems to create projects